manga_translator/inpainting/inpainting_lama_mpe.py

```
@@ -575,7 +588,10 @@ async def _load(self, device: str, force_torch: bool = False):
             self._downloaded = True
         
         # 直接加载到目标设备，避免重复移动
-        target_device = device if (device.startswith('cuda') or device == 'mps') else 'cpu'
+        target_device = self.torch_device if (device.startswith('cuda') or device == 'mps') else 'cpu'
+        if device == 'dml':
+            # PyTorch torch-directml 不支持 ComplexFloat 复数算子，在 PyTorch 模式下强制使用 CPU 运行以确保 100% 稳定
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)